region/wreckedship/main.json

```diff
@@ -2121,6 +2121,30 @@
                 }
               ],
               "openEnd": 1
+            },
+            {
+              "name": "Full Runway - Bowling Alley Top Left Door (to West Ocean Upper)",
+              "length": 33,
+              "strats": [
+                {
+                  "name": "Conveyor Powered On",
+                  "notable": false,
+                  "requires": ["f_DefeatedPhantoon"]
+                },
+                {
+                  "name": "Shinespark to Clear Speed Blocks",
```

Comment:
could be worthwhile adding a strat coming in blue, now that we are using more runways that have unused tiles (speed balls, etc)

Reply:
I'm not sure how `unusedTiles` would resolve because a `canComeInCharged` would be referring to a runway blocked by the `canComeInCharge`.  Or it may first try to use the shorter runway with no requirements.

And being able to enter with blue already means you can leave with blue.

Reply:
I'm referring to three-room strats (which arent able to be used with the current schema yet, but hopefully we can soonish).

One example is if Phantoon is not defeated and this is connected to Door 6 of WS Main Shaft. You have 31 tiles coming in (3+28) and then performing a speed ball to the left puts you at 17 tiles leaving (3+28-14), because of the `unusableTiles`.